### ISSUE TITLE
feature/v2 regression-metrics: MSE

### ISSUE THREAD

**Issue by JasonShin** [kalimdorjs/kalimdorjs#136]
* **I'm submitting a ...**
[/] feature request

* **Summary**

The mean squared error tells you how close a regression line is to a set of points. It does this by taking the distances from the points to the regression line (these distances are the “errors”) and squaring them. The squaring is necessary to remove any negative signs. It also gives more weight to larger differences. It’s called the mean squared error as you’re finding the average of a set of errors.

Introducing MSE will be beneficial for the users who are using Regression models such as RandomForestRegressor or SGDRegressor, and they need to check the prediction accuracy. In addition, it will be useful for unit testing purposes.

Reference:
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html

**Comment by JasonShin** [comment]
I am just going to use https://js.tensorflow.org/api/latest/#losses.meanSquaredError

**Comment by JasonShin** [comment]
This is merged into V2
